rule_packages/c/Pointers3.json

```diff
@@ -72,6 +72,7 @@
           "precision": "medium",
           "severity": "error",
           "short_name": "DoNotPassAliasedPointerToRestrictQualifiedParam",
+          "shared_implementation_short_name": "DoNotPassAliasedPointerToRestrictQualifiedParam_Shared",
```

Comment:
Our other shared queries typically use "Shared" as a suffix as opposed to "_Shared," if there is any suffix at all.